bpm.sh

```diff
@@ -165,10 +165,10 @@ __bpm_list_enabled_by_deps() {
     (
     [[ -d "$BPM_HOME"/enabled ]] || return 1
     cd "$BPM_HOME"/enabled >/dev/null
-    local latest=$(command ls -tdL . * | head -n 1)
+    local latest=$(command ls -tdL . ./* 2>/dev/null | sed -n 2p)
     # echo $latest >&2
     deps="$BPM_TMPDIR"/enabled.deps
-    if [[ "$deps" -nt $latest ]]; then
+    if [[ -h $latest && "$deps" -nt $latest ]]; then
```

Comment:
Not sure I follow this part.  Let me take another stab.  I think this was to detect any `bpm on/off` as well as enabled plugins getting updated since enabled.deps was last generated.

Hope you don't mind my pushing commits to this PR :)